ORBIT/config.py

```diff
@@ -1,10 +1,12 @@
+"""Provides the configuration loading and saving methods."""
+
 __author__ = "Jake Nunemaker"
 __copyright__ = "Copyright 2020, National Renewable Energy Laboratory"
 __maintainer__ = "Jake Nunemaker"
 __email__ = "jake.nunemaker@nrel.gov"
 
 
-import os
+from pathlib import Path
```

Comment:
It's been generally suggested since ~3.7 to switch over to the use of `pathlib.Path` for managing file paths because it's a cleaner and easier to read interface. That said, we don't have to, and most of those changes are in a single commit.